test/test_partshop.py

```diff
@@ -213,20 +213,63 @@ def test_uri2url_spoofed(self):
 
     @unittest.skipIf(password is None, "No password supplied")
     def test_attach_file(self):
+        sbol2.Config.setOption('sbol_typed_uris', False)
         doc = sbol2.Document()
         doc.displayId = 'test_attachment'
         doc.name = 'test attachment'
         desc = 'a test collection created by the sbol2 unit tests'
         doc.description = desc
         md = doc.moduleDefinitions.create('attachmd')
+        # attachment = doc.attachments.create('attachment')
+        # attachment.source = CRISPR_LOCATION
+        # attachment.format = 'http://purl.org/NET/mediatypes/application/xml'
+        # md.attachments = [attachment.identity]
+        # sbh = sbol2.PartShop(TEST_RESOURCE)
+        # sbh.login(username, password)
+        # sbh.submit(doc, overwrite=1)
+        # attachment_uri = '{}/user/{}/{}/{}/{}'.format(sbh.getURL(), sbh.getUser(),
+        #                                               doc.displayId,
+        #                                               attachment.displayId,
+        #                                               attachment.version)
```

Comment:
Do we need all of this commented out code?

Reply:
Cleaned up